.github/workflows/docker_build_push.yml

```diff
@@ -94,25 +38,3 @@ jobs:
           ref: main
           wait_interval: 60
           client_payload: '{ "topos-docker-tag": "${{ env.docker_tag }}" }'          
-
-  frontend-erc20-e2e:
```

Comment:
When does the `erc20-e2e` gets run now ?

Reply:
[Here](https://github.com/topos-protocol/topos/actions/runs/7273739129/job/19818398504?pr=397)
[This workflow] (https://github.com/topos-protocol/e2e-tests/actions/workflows/topos:integration-tests.yml)

Reply:
I don't see where it doesn't trigger the `frontend` [e2e tests](https://github.com/topos-protocol/e2e-tests/actions/runs/7273564155), it only run the new set of scripts.

Reply:
It now runs e2e `topos:integration-tests.yml` instead of `frontend:erc20-messaging.yml`.  Execution of `topos:integration-tests.yml` was added in one of previous PRs

Reply:
Here are the e2e workflow [definitions](https://github.com/topos-protocol/e2e-tests/tree/main/.github/workflows)

Reply:
Yes but we need to keep executing `frontend:erc20-messaging.yml`

Reply:
No the point of this work was to remove dependency of frontend from topos PRs. We do all relevant tests in `topos:integration-tests.yml`, frontend tests are to be executed as part of frontend and smart contract checks

Reply:
@atanmarko The initial point was to add more tests **_aside_** of the frontend e2e tests. Not to replace it, we still want to run it.

But just to understand your point, you believe that running this workflow for `topos` PRs doesn't make sense because never a PR on `topos` would make `topos:integration-tests.yml` pass but break `frontend:erc20-messaging.yml`?

Reply:
@hadjiszs Right, `frontend:erc20-messaging.yml` tests sending cross chain transaction. It uses topos protocol to pass certificate from one subnet to another, that that is what we are testing in `topos:integration-tests.yml` in multiple ways. We should not break topos regression tests because of some incompatibility in frontend/smart contract interactions.

Reply:
If you still think that we should run it I'll return frontend tests in [PR399](https://github.com/topos-protocol/topos/pull/399)

Reply:
I see the point, that said I'm not 100% sure that everything `topos` related that is tested by `frontend:erc20-messaging.yml` is really covered at 100% by `topos:integration-tests.yml`, now and in the future.

Overall, I'd feel more safe to keep running it for all `topos` PRs aside of the new ones, so yep I'm happy if you can bring it back in #399